app/controllers/books_controller.rb

```diff
@@ -6,11 +6,21 @@ def book
     @book = Book.find(params[:id])
   end
 
+  def filter
+    @book_search = Book.where(nil)
```

Comment:
I'd just use `Book` here since render_paginated admits models as well (try it before pushing though)